Core/GDCore/Extensions/Builtin/SceneExtension.cpp

```diff
@@ -18,7 +18,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsSceneExtension(
           _("Built-in extension allowing to manipulate scenes and providing common features"),
           "Florian Rival",
           "Open source (MIT License)")
-      .SetExtensionHelpPath("" /*TODO: Add a documentation page for this */);
+      .SetExtensionHelpPath("/interface/scene-editor/events");
```

Comment:
Same here, be careful because doing this you'll be adding a help page for expressions like Random (they might not be shown in the editor, but one day they will, and in this case we would have wrong link for RandomXXX expressions, which is worse than no link).
Instead, use SetHelpPath to set the help paths for each condition/action.